design/ux/Obscuro_Gateway.md

```diff
@@ -129,7 +129,39 @@ Alice -> OG: All further Obscuro interactions will be to\nhttps://gateway.obscur
 The onboarding should be done in 3 clicks.
 1. The user goes to a website (like "obscuro.network"), where she clicks "Join Obscuro". This will add a network to their wallet.
 2. User connects the wallet to the page.
-3. In the wallet popup, the user has to sign over a message: "Register $UserId for $ACCT"
+3. In the wallet popup, the user has to sign over EIP-712 formatted message.
+
+Format of EIP-712 message used for signing viewing keys is:
+
+```
+types: {
+      EIP712Domain: [
+        { name: "name", type: "string" },
+        { name: "version", type: "string" },
+        { name: "chainId", type: "uint256" },
+        { name: "verifyingContract", type: "address" },
+      ],
+      Register: [
+        { name: "userID", type: "string" },
+        { name: "account", type: "address" },
+      ],
+    },
+    primaryType: "Register",
```

Comment:
is the primaryType displayed somewhere?

Reply:
<img width="346" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/4050716/1dd266a0-a13e-420f-a029-3cc1ac0d5028">

It is displayed here. Do you suggest another name?

Reply:
@CaisManai , wdyt?

Reply:
After changing userID to address it looks like this:
<img width="352" alt="image" src="https://github.com/ten-protocol/go-ten/assets/4050716/2a0f6a23-38f2-49bd-be50-25e5ac4dca40">

Reply:
Me and Tudor did some brainstorming. I think we should replace UserID with either:
- Encrypted Account
- Encrypted Address

Reply:
![image](https://github.com/ten-protocol/go-ten/assets/4050716/de973c0a-6bad-4790-ac62-2ba06fe417e8)

fixed, looks much better and less scary to sign 👍